packages/logs/src/boot/logsPublicApi.ts

```diff
@@ -32,6 +32,7 @@ const LOGS_STORAGE_KEY = 'logs'
 
 export interface Strategy {
   init: (initConfiguration: LogsInitConfiguration) => void
+  setTrackingConsent: StartLogsResult['setTrackingConsent']
```

Comment:
<!-- Warning represent user-facing problems. If possible, it's great to follow this kind of comment with a suggestion.  -->
**❗ warning:** ‏the API also needs to be exposed from the global variable.

Reply:
Yeah I figured it out 😄